### ISSUE TITLE
Add translation for player name change product page

### ISSUE THREAD

**Issue by rezemoe** [ppy/osu-web#6152]

**Comment by peppy** [comment]
Do you mean for item descriptions? If so, this is not a simple one due to them being databased.

**Comment by lmnyx** [comment]
Yeah, but atleast name changing page can be easily added I think?
![p_n_c](https://user-images.githubusercontent.com/13962537/84328592-cdd83280-ab9b-11ea-8217-8747d67f93d9.png)

**Comment by peppy** [comment]
Username change specifically can probably be done, yes.
